### PR TITLE
Update configuration_shippingpackaging.md

### PR DESCRIPTION
**Pull request by brittainmark**
Added extra example to show that decimal values are OK on SHIPPING_BOX_WEIGHT and SHIPPING_BOX_PADDING. Added Footnote for SI units (kg),
Changed the wording for SHIPPING_WEIGHT_UNITS as it implies that you should only use lbs.

Don't know if descriptions should also be changed in zencart

### CODE REVIEW

**Comment by drbyte** [comment]
@scottcwilson I think this page is auto-generated from configuration table entries, isn't it? 
Might have to forego the footnote in that case.

@brittainmark if we just add the fraction example into the core SQL, that's probably fine. 
I'm impartial to whether it needs a kgs example, but I suppose one could be included instead of footnoting.

Also, I guess the note about 1oz=0.0625 lbs could be considered optional. I'm impartial.

**Comment by brittainmark** [comment]
Accepted. Do you want me to try to word something for the install files.

**Comment by drbyte** [comment]
> Accepted. Do you want me to try to word something for the install files.

Yes please.

**Comment by brittainmark** [comment]
How about rewording the example
Example:
Unt = your shipping unit (lbs or kgs)
10% + 1Unt 10:1
10% + 0Unt 10:0
 0% + 5Unt 0:5
 0% + 1/2Unt 0:0.5
 0% + 0Unt 0:0

**Comment by drbyte** [comment]
Ya, that's a good way to make it universal.

I wonder if maybe better to use the full word `unit` though?

**Comment by brittainmark** [comment]
Created pull request on zen cart #6261
